.paket/Paket.Restore.targets

```diff
@@ -0,0 +1,276 @@
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
```

Comment:
What's this file all about?

Reply:
It looks like this file is needed for proper paket support in VS 2017. See this thread: https://github.com/fsprojects/Paket/issues/2840